.gitignore

```diff
@@ -6,8 +6,16 @@ noBackup/
 *.qml.autosave
 .vscode
 bin/
+pkg/
 vendor/.nimble
 *.AppImage
 tmp
 nimcache
-.DS_Store
\ No newline at end of file
+.DS_Store
+node_modules
+npm-debug.log*
+npm-shrinkwrap.json
+package
+yarn-debug.log*
+yarn-error.log*
+yarn.lock
```

Comment:
I see you put `package.lock` here in gitignore, but it is still pushed below. Does one of them need to be removed?

Reply:
There's a line in `.gitignore` for `/package/`, which is just in case someone runs `npm pack` (pointless but might happen 🤷‍♂️ ) and then unpacks the tarball in the root of the repo, the contents of which would expanded into a directory named `package`.